Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-AKS-cluster/sample-workload-for-agic-test.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
```

Comment:
Please indicate at the top of this file that this is only used for the B/G scenario

Reply:
fixed